src/creation.ts

```diff
@@ -27,7 +28,13 @@ task("create", "Create a Safe")
         )
         const predictedAddress = await calculateProxyAddress(factory, singleton.address, setupData, taskArgs.nonce)
         console.log(`Deploy Safe to ${predictedAddress}`)
-        await factory.createProxyWithNonce(singleton.address, setupData, taskArgs.nonce).then((tx: any) => tx.wait())
+        console.log(`Singleton: ${singleton.address}`)
+        console.log(`Setup data: ${setupData}`)
+        console.log(`Nonce: ${taskArgs.nonce}`)
+        console.log(`To (factory): ${factory.address}`)
+        console.log(`Data: ${factory.interface.encodeFunctionData("createProxyWithNonce", [singleton.address, setupData, taskArgs.nonce])}`)
```

Comment:
Should place this log inside the `if` and extract `factory.interface.encodeFunctionData("createProxyWithNonce", [singleton.address, setupData, taskArgs.nonce])` into a value to avoid running the encoding twice?